Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -6,6 +6,7 @@ Authors: Reid Barton, Mario Carneiro, Isabel Longbottom, Kim Morrison, Yuyang Zh
 import Mathlib.Algebra.Order.ZeroLEOne
 import Mathlib.Data.List.InsertIdx
 import Mathlib.Logic.Relation
+import Mathlib.Logic.Relator
```

Comment:
Surely this import isn't needed if you're already working with `Relator.BiTotal`, right?